libyul/Object.cpp

```diff
@@ -88,6 +91,35 @@ string Object::toString(
 	return useSrcComment + "object \"" + name.str() + "\" {\n" + indent(inner) + "\n}";
 }
 
+Json::Value Data::toJson() const
+{
+	Json::Value ret{Json::objectValue};
+	ret["nodeType"] = std::move("YulData");
+	ret["value"] = util::toHex(data);
+	return ret;
+}
+
+Json::Value Object::toJson() const
+{
+	yulAssert(code, "No code");
+	yulAssert(debugData, "No debug data");
+
+	Json::Value code_json{Json::objectValue};
+	code_json["nodeType"] = std::move("YulCode");
```

Comment:
Is there any point in moving string literals? :)
```suggestion
	code_json["nodeType"] = "YulCode";
```